### PR TITLE
STACCoverflow validated-tokens.csv Community Claim

### PR DESCRIPTION
**Pull request by BlockchainSavingsNonprofitBank**
A community claimed ownership for this token on May 17 2024

This is the Community Claim (the initiator has departed). Token and market were the first STACC ever launched on Raydium after the historical episode at pump.fun:
https://dexscreener.com/solana/5uqdg5osqmvzpvihr95fw2hqzekxsusexfggvj2pf6nt
Holders, makers: 1230.
The token was the first STACC or staccoverflow launched on Raydium after the historical episode at http://pump.fun:  https://x.com/STACCoverflow/status/1791134470071865576 - the pump.fun exploiter

# Validate [STACCOVERFLOW](https://solscan.io/token/4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4)

## Attestations (Please provide links):
- [x] Tweet from your Twitter Account attesting the Mint address https://x.com/staccsonsolana
 https://x.com/staccsonsolana/status/1791407285471736162?s=46&t=5uOjKiKUZdUsVIszHn-3gg
- [x]  Coingecko/ CMC URL (If available): https://www.geckoterminal.com/solana/pools/5UQDg5osqmVZPvihr95Fw2hqZEKxSuSEXFgGVj2Pf6NT

https://solscan.io/account/5UQDg5osqmVZPvihr95Fw2hqZEKxSuSEXFgGVj2Pf6NT
https://solscan.io/token/4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4
https://dexscreener.com/solana/5uqdg5osqmvzpvihr95fw2hqzekxsusexfggvj2pf6nt

https://x.com/staccsonsolana or https://x.com/StaccArmy

## Validation (Please check off boxes):
- [x] [Yes!] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] [Yes!] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] [Yes!] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
 - [x] [Yes!] My change matches the format in the file (no spaces between fields).
 - [x] [Yes!] My token is already live and trading on Jupiter.
 - [x] [Yes!] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
 - [x] [Yes!] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

STACCOVERFLOW,STACC,4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4,6,https://dd.dexscreener.com/ds-data/tokens/solana/4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4.png,true

 - [x] LIQUIDITY LOCKED & BURNED
FDV
$61K
MKT CAP
$61K

{
name:
"STACCOVERFLOW"
symbol:
"STACC"
description:
"staccoverflow hacker behind infamous pump.fun crash"
image:
"https://tan-thirsty-wolf-360.mypinata.cloud/ipfs/QmQkKLqjN1FFCrLZLn5zvs8jiNvbtJmXxuiQWGhiv49bU6"
address: "4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4"
}
}

- [ ] Some members of the community who were a little late joined another STACC. It's not the original one, but you can verify them too; their symbol is different. To prove that STACC, Staccoverflow, is the first and original one, there is a screeshoot attached.
The second STACC doesn't claim to be the original "overflow", so they are legitimate too.
- [x] Permanently saved proof: https://arweave.net/9zb-yXSo3l9RlS7VHzHNePjdKOz7LC2n_Jp7ptwFoio 17 May 2024

### CODE REVIEW

**Comment by BlockchainSavingsNonprofitBank** [comment]
alternative image: image:
"https://dd.dexscreener.com/ds-data/tokens/solana/4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4.png?size=lg&key=d75359"

**Comment by BlockchainSavingsNonprofitBank** [comment]
https://x.com/staccsonsolana/status/1791407285471736162?s=46&t=5uOjKiKUZdUsVIszHn-3gg

community post

**Comment by BlockchainSavingsNonprofitBank** [comment]
Okay, some members of the community who were a little late joined another STACC. It's not the original one, but you can verify them too; their symbol is different. To prove that STACC, Staccoverflow, is the first and original one, there is a screeshoot attached.
 The second STACC doesn't claim to be the original "overflow", so they are legitimate too.

Permanently saved proof: https://arweave.net/9zb-yXSo3l9RlS7VHzHNePjdKOz7LC2n_Jp7ptwFoio

image (1)

**Comment by BlockchainSavingsNonprofitBank** [comment]
The origin : https://x.com/STACCoverflow/status/1791134470071865576

File: validated-tokens.csv [hunk 1]
```diff
@@ -1573,3 +1573,4 @@ People of Walmart,POW,wA1MArtTmL1yYtweBjyHZZdteHQRd5fgwsUKx7iiqKU,6,https://arwe
 Lets Fuckin Go,LFGO,2u98MM7DMtVmNG4iAKRNMtynjmkzgD6fXAzB3wVfhQvg,6,https://bafybeihigmjodulnlbn6dve7nmmwpkdgno5beid3hx5ku5irfy4wjmredy.ipfs.cf-ipfs.com/,true
 Drift,DRIFT,DriFtupJYLTosbwoN8koMbEYSx54aFAVLddWsbksjwg7,6,https://metadata.drift.foundation/drift.png,true
 steep jubs,opple,BG5Dp9gU5WbkHEaz6y95apb5NVPw3jC17M4ro27sgSXP,9,https://bafkreig7kb2jifavh2buo4l35op6dapjuunozaiwves42qgla5pk54t7ki.ipfs.nftstorage.link/,true
+STACCOVERFLOW,STACC,4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4,6,https://tan-thirsty-wolf-360.mypinata.cloud/ipfs/QmQkKLqjN1FFCrLZLn5zvs8jiNvbtJmXxuiQWGhiv49bU6,true
```
Comment (BlockchainSavingsNonprofitBank) [inline validated-tokens.csv:1576]: STACCOVERFLOW,STACC,4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4,6,https://dd.dexscreener.com/ds-data/tokens/solana/4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4.png,true

**Review by BlockchainSavingsNonprofitBank — COMMENTED** [review]
pls update the image if you want

**Comment by BlockchainSavingsNonprofitBank** [comment]
https://rugcheck.xyz/tokens/4FaWA6N2WEdfKS14xGGfwd99TaNg8BJB5xSsrNdPHmU4
Risk Analysis
500
Good

**Comment by Jymfletzfur** [comment]
insufficient volume

**Pull request closed without merging**